examples/morphir-elm-projects/evaluator-tests/src/Morphir/Examples/App/ExceptionTests.elm

```diff
@@ -27,3 +28,55 @@ decimalHundred value =
 ignoreArgReturnString : a -> String
 ignoreArgReturnString _ =
     "test"
+
+
+{-| Test: ExceptionTests/acceptTuple3
+-}
+acceptTuple3 : ( Int, Int, Int ) -> String
+acceptTuple3 _ =
+    "accept tuple test"
+
+
+type alias XYRecord =
+    { x : Int, y : Int }
+
+
+{-| Test: ExceptionTests/accept XYRecord
+-}
+acceptRecord : XYRecord -> String
+acceptRecord _ =
+    "accept record test"
+
+
+type alias AliasAlias =
```

Comment:
This name concerns me - the name (to me) implies it's an alias to an alias, but this is an alias to a record type which contains an alias to another record type. If you want an alias to an alias, that would just be `type alias AliasAlias = XYRecord`. Maybe change the name if you wanted this to just be a nested record thing?

Reply:
good point. Some of the definitions for these changed as I was working on it, so I'll recheck the names generally